news/2024/2024-03-19-bespoke-music-in-osu.md

```diff
@@ -0,0 +1,259 @@
+
+[![](/wiki/shared/news/2024-03-19-bespoke-music-in-osu/takehirotei.png)](https://osu.ppy.sh/users/11793794)
+
+> \[...] I will be sharing one of the memories that has left a lot of impact in my heart. The first one is during the process of making [Essence](https://osu.ppy.sh/beatmapsets/1943786) with HowToPlayLN. This is my first time collaborating with someone (as in 2 ppl collab) on making a Grand Finals tiebreaker. At first, we are just trying to put in the tiebreaker copypasta into a tiebreaker song, but as time goes by, our ideas become crazier and crazier. At one point we even decide to make a section where the time signature changes every bar.
+>
+> After the tiebreaker was made, things got even more crazy as the media team is cooking the lore. At first we only have the tiebreaker, then soon enough, we all decide to make a song for each cutscene that shows before the mappool showcase begins, and also a [rearrangement of *Essence* for the credits theme](https://www.youtube.com/watch?v=YcTvEi8ZPyA). This is perhaps the most precious memories that I have ever been through at that point as a composer.
+
+[![](/wiki/shared/news/2024-03-19-bespoke-music-in-osu/HowToPlayLN.png)](https://osu.ppy.sh/users/10879600)
+
+> There are so many moments in music journey which reminded me that my life has meanings. Some other people would tell me directly that my compositions are emotionally touching to them, like *Akatsuki*, or *Essence*, or some actually perceived them as "insane". Especially if those are meant to be portraying my emotions at the time I was composing them, I would feel grateful a lot. It feels like somebody understood me and my intent of doing my best in my own hobby.
+> 
+> The most significant ones would be *Essence*, which I considered one of the significant moments of my life at that time. It was the biggest and the most emotional project we have ever done. I cannot stop going back and watch the reactions of the viewers at that time to the reveal of the tiebreaker song... It was both insane and emotional, and was the conclusion of the most productive session I've done at that time.
+> 
+> The other one which probably less or more related which is [osu!mania LN Tournament 3](https://www.youtube.com/watch?v=YcTvEi8ZPyA), I also did more than 10 custom maps for that tournament, including the custom song "[LNCryption](https://osu.ppy.sh/beatmapsets/1893946)", which was actually the space for the mappers to write and imagine the "hell" they faced somewhere else. It probably is worth to give them a read.
+> 
+> I would also give an honorable mention to [rememberance; image of him before departure](https://osu.ppy.sh/beatmapsets/2028818) from Vietnamese National osu!mania Cup, which I composed in one day, during one of my depressive episodes. It was meant to be something that is lighthearted and healing me up. Quick compositions like this can also help me coping through hard times as well.
```

Comment:
```suggestion
> I would also give an honorable mention to [remembrance; image of him before departure](https://osu.ppy.sh/beatmapsets/2028818) from Vietnamese National osu!mania Cup, which I composed in one day, during one of my depressive episodes. It was meant to be something that is lighthearted and healing me up. Quick compositions like this can also help me coping through hard times as well.
```